lib/store.js

```diff
@@ -17,6 +41,151 @@ const setCpuGridChoice = (store, choice) => {
   store.choices.push(choice);
 };
 
+const setWinner = (store) => {
+  store.winner = true;
+};
+
+//--------------- inline store gettter/setters --------------//
+const store1 = {
+  _grid: {
+    1: 1,
+    2: 2,
+    3: 3,
+    4: 4,
+    5: 5,
+    6: 6,
+    7: 7,
+    8: 8,
+    9: 9,
+  },
+  _userTeam: null,
+  _cpuTeam: null,
+  _firstPlayer: null,
+  _choices: [],
+  _winner: false,
+  get grid() {
+    return this._grid;
+  },
+  set grid(key, value) {
```

Comment:
How does `value` get passed in here? My instincts expect it to always be `undefined`